libs/components/modals/src/lib/modules/modal/modal-footer.component.ts

```diff
@@ -9,4 +12,14 @@ import { Component, ViewEncapsulation } from '@angular/core';
   styleUrls: ['./modal-footer.component.scss'],
   encapsulation: ViewEncapsulation.None,
 })
-export class SkyModalFooterComponent {}
+export class SkyModalFooterComponent {
+  public formErrors: SkyModalError[] | undefined;
+
+  #adapterSvc = inject(SkyModalComponentAdapterService);
+
+  constructor() {
+    this.#adapterSvc.formErrors.subscribe((errors) => {
```

Comment:
I'd prefer to see the service's `formErrors` property referenced directly in the template using the `async` pipe. That way it will be unsubscribed when the component is destroyed, avoids the "magic" of Angular change detection picking up the change to `formErrors` outside of an input changing, and eliminates a decent amount of code from the class.

Template:

```
  <div aria-live="polite">
    <div *ngIf="(errorsSvc.formErrors | async) as formErrors" class="sky-modal-footer-errors">
      <sky-status-indicator
        *ngFor="let formError of formErrors"
        descriptionType="error"
        indicatorType="danger"
      >
        {{ formError.message }}
      </sky-status-indicator>
    </div>
  </div>
```
Class:

```
  protected errorsSvc = inject(SkyModalErrorsService);
```

Reply:
added